python/pyproject.toml

```diff
@@ -30,6 +30,13 @@ dependencies = [
   "pydantic>=2.10.0",
   "kubeflow_trainer_api@git+https://github.com/kubeflow/trainer.git@master#subdirectory=api/python_api"
 ]
+[project.optional-dependencies]
+test = [
+  "pytest>=7.0",
+  "pytest-mock>=3.10",
+  "coverage>=7.0",
+  "coveralls>=4.0.1",
```

Comment:
We we require coveralls as test dependency if we only use it in GitHub actions to send report ?
Maybe we can consider to use the official Coveralls GitHub action ?
https://github.com/marketplace/actions/coveralls-github-action

Reply:
done